filesystem/fs.go

```diff
@@ -76,11 +76,13 @@ func (cfs CFS) create(parentID uint64, name string, node *Node) error {
 	if err != nil {
 		return err
 	}
-	const sql = `
-INSERT INTO fs.inode VALUES ($1, $2);
-INSERT INTO fs.namespace VALUES ($3, $4, $1);
-`
-	if _, err := tx.Exec(sql, node.ID, inode, parentID, name); err != nil {
+	const insertNode = `INSERT INTO fs.inode VALUES ($1, $2)`
+	if _, err := tx.Exec(insertNode, node.ID, inode); err != nil {
+		_ = tx.Rollback()
+		return err
+	}
+	const insertNamespace = `INSERT INTO fs.namespace VALUES ($1, $2, $3)`
+	if _, err := tx.Exec(insertNamespace, parentID, name, node.ID); err != nil {
```

Comment:
Cargo culting from [stack overflow](http://stackoverflow.com/questions/20561254/insert-data-in-3-tables-at-a-time-using-postgres):

```
WITH a AS (
    INSERT INTO t1 ...
),
b AS (
    INSERT INTO t2 ...
)
SELECT 1;
```